### DISCUSSION TITLE
How can it be faster?

### DISCUSSION THREAD

**Discussion by hebo1982** [coqui-ai/TTS#511] [General]
I try tts() on GPU, 1080TI.
TTS 0.0.13
with model:
tts_models--en--ljspeech--tacotron2-DDC
vocoder_model--en--ljspeech-hifigan_v2

output:
['TTS is a library for advanced Text-to-Speech generation.', "It's built on the latest research, was designed to achieve the best trade-off among ease-of-training, speed and quality.", 'TTS comes with pretrained models, tools for measuring dataset quality and already used in 20+languages for products and research projects.']
 > Processing time: 4.243091821670532
 > Real-time factor: 0.619849083169852

That  can be faster?

**Comment by domcross** [comment]
In case you want to generate a lot of phrases you may try tts-server, this saves the time for loading and initializing the tts-model.

And faster hardware (e.g. RTX3090) always helps ;-)

**Reply by hebo1982** [reply]
:P 3090
